assets/js/incentiveLevel.js

```diff
@@ -0,0 +1,14 @@
+$(function () {
+  // record incentive level confirmation page
+  $('.govuk-button').on('click', function (e) {
+    const $prompt = $(e.target)
+    const gaId = $prompt.data('ga-id')
+    const caseload = $prompt.data('case-load')
+    if (gaId && typeof gtag === 'function') {
+      gtag('event', 'click', {
+        event_category: gaId,
+        event_label: caseload,
+      })
```

Comment:
the particular details of the event sent to google analytics need to be adapted to make sense in the context of _this_ app. these likely made sense on dps, but not so much here cos different names are used.

so basically, the purpose is to track which of the 2 links are clicked on the confirmation page. what i'd do is adapt this `gtag` call to be more like what’s in `/assets/js/gaSendEvents.js#L20`

1) the active case load is already set globally so you can get rid of the `data-case-load` attribute in the template and the js code here that pulls it out
2) i would hazard a suggestion to rename the event from `click` to `incentives_event` to match the other events sent by this site. @xoen do you agree? using the same name would make selecting and filtering easier i presume.
3) instead of `event_category: gaId,` we should have 2 rows with [category and label](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/assets/js/gaSendEvents.js#L21-L22) – where the `category` should be the _same_ for both buttons (something like "Record incentive level confirmation") and the `label` is what distinguishes them (e.g. "manage reviews" and "case notes")

Reply:
yes, you could track this by binding the on click event like done in here: https://github.com/ministryofjustice/hmpps-incentives-ui/blob/9f8be5b8ff32678665b4ecbccbc612e34c80200b/assets/js/gaSendEvents.js#L6